internal/provider/kubernetes/controller.go

```diff
@@ -1578,14 +1578,14 @@ func (r *gatewayAPIReconciler) processParamsRef(ctx context.Context, gc *gwapiv1
 	}
 
 	epList := new(egv1a1.EnvoyProxyList)
+	gcParametersRefNamespace := string(*gc.Spec.ParametersRef.Namespace)
```

Comment:
nil check ?

Reply:
nvm its checked in `refsEnvoyProxy` https://github.com/envoyproxy/gateway/blob/516a27d6d7d8d35686fc7b8391864b123aaea7f7/internal/provider/kubernetes/helpers.go#L195